styles.css

```diff
@@ -94,19 +94,35 @@ li.century {
     border: 2px solid black;
     box-shadow: 5px 3px 3px #3f2a14;
     background-color: white;
-    height: 40vh;
+    min-height: 29vw;
     width: 23vw;
     margin-bottom: 1%;
     cursor: pointer;
 }
 
+h3 {
```

Comment:
Added some styling for the card text so it will change based on the screen size!